controller/work_item_relationships_links.go

```diff
@@ -24,6 +24,7 @@ type WorkItemRelationshipsLinksController struct {
 // WorkItemRelationshipsLinksControllerConfig the config interface for the WorkItemRelationshipsLinksController
 type WorkItemRelationshipsLinksControllerConfig interface {
 	GetCacheControlWorkItemLinks() string
+	GetCacheControlWorkItemLink() string
```

Comment:
so this one looks unnecessary, since there's no `Show` operation for the work item relationship links ?